docs/getting_started.md

```diff
@@ -59,6 +59,18 @@ raw_counts <- counts(processed_sce)
 normalized_counts <- logcounts(processed_sce)
 ```
 
+If you have ADT data from a CITE-seq experiment, you can use the following commands to access raw and normalized ADT count matrices:
+
+```r
+# the raw ADT counts matrix stored in the processed object
+raw_adt_counts <- counts(altExp(processed_sce))
+
+# log normalized ADT counts matrix stored in the processed object
+# if this value is NULL, it means normalization failed
+normalized_adt_counts <- logcounts(altExp(processed_sce))
+```
```

Comment:
I think I would prefer to put everything CITE-seq related into a separate section for working with CITE-seq data. So I would move this to the new section you added. There you would include this and then move onto filtering and normalizing.